modular_chomp/code/modules/clothing/glasses/glasses.dm

```diff
@@ -6,3 +6,28 @@
 	icon_state = "AR_visor_tesh"
 	item_state = "AR_visor_tesh"
 	sprite_sheets = list(SPECIES_TESHARI = 'icons/inventory/eyes/mob_teshari.dmi')
+
+
+/obj/item/clothing/glasses/darksight
+	name = "voidviewer goggles"
+	desc = "Cryptic goggles designed for seeing beyond the viel of normality."
+
+	icon = 'modular_chomp/icons/inventory/eyes/glasses.dmi'
+	icon_override = 'modular_chomp/icons/inventory/eyes/mob.dmi'
+	icon_state = "darksight"
+
+	action_button_name = "Toggle Goggles"
+
+	toggleable = 1
+	enables_planes = list(INVISIBILITY_SHADEKIN)
+
```

Comment:
This is not a plane.

~~enables_planes = list(INVISIBILITY_SHADEKIN)~~


see_invisible = INVISIBILITY_SHADEKIN